hardware/include/cs_io_if.vh

```diff
@@ -0,0 +1,6 @@
+    //control-status i/o (do not remove indentation)
```

Comment:
File not needed. Signals shoul be at top

Reply:
Should these signals be removed from the top's interface signals?